### PR TITLE
Banana Slippers

### PR DESCRIPTION
**Pull request by Kingsley-95**
## About The Pull Request
The Slipers (the single p is purposeful). 

A new clothing item takes a similar role to a bola. When thrown, if aiming at either leg, it will try to knock off the shoes of whoever is it, then, equip onto them. Once equipped, they will be unable to take a step without being slipped. Additonally, if they are slipped, anybody who walks over them will ALSO be slipped. And the item itself is slippery. Because they are banana shoes, get it?

Finally, if hit with these banana boots, the wearer is unable to remove them, someone will need to strip them off.

## Why It's Good For The Game
More variety in trouble making tools that isnt another shoot shoot bang bang. 

## Proof Of Testing
https://youtu.be/jyodCikoiXw

<details>
<summary>Screenshots/Videos</summary>

</details>

## Changelog

:cl:
add: Slipers, a new traitor item which can be bought from the uplink
/:cl:

### CODE REVIEW

**Comment by NargaInDisguise** [comment]
You should make these clown restricted, it'd fit 'em and give them more funny tools.

Also, personally I believe that it should have a solo resist timer to take off, so that people can't suffer for too long if they somehow escape and people don't help them. Maybe 30 seconds, a minute?

**Comment by Kingsley-95** [comment]
> You should make these clown restricted, it'd fit 'em and give them more funny tools.
> 
> Also, personally I believe that it should have a solo resist timer to take off, so that people can't suffer for too long if they somehow escape and people don't help them. Maybe 30 seconds, a minute?

Made it clown restricted, great point. As for the second bit, I viewed it like a less punishing glue / cursed item. So, rather than needing to cut the limb off entirely, only another player is needed to help get them off. Otherwise, it'd just be an expensive bola considering the TC requirements. Just my thoughts on it, happy to discuss it more

**Comment by Arturlang** [comment]
A idea might be is making it craftable out of bananium and clown shoes

**Comment by lessthnthree** [comment]
- Is there a reason it's priced high as 6 TC? Looking at other job items it seems like it'd be more like 3.
- Is calling it 'slipers' instead of 'slippers' intentional?

File: modular_zubbers/code/modules/clothing/shoes/boots.dm [hunk 1]
```diff
@@ -190,3 +190,78 @@
...
+/obj/item/clothing/shoes/slipers/throw_impact(atom/hit_atom, datum/thrownthing/thrownthing)
+	. = ..()
+	///if the thrown object's target zone isn't the head
+	if(thrownthing.target_zone != BODY_ZONE_L_LEG && thrownthing.target_zone != BODY_ZONE_R_LEG)
+		return
+	///Just in case someone adds storage down the line on the slipers
+	if(LAZYLEN(contents))
+		return
+	if(iscarbon(hit_atom))
+		var/mob/living/carbon/H = hit_atom
```
Comment (lessthnthree) [inline modular_zubbers/code/modules/clothing/shoes/boots.dm:215]:
```suggestion
		var/mob/living/carbon/hit_carbon = hit_atom
```
No one/two letter vars, please.

Comment (lessthnthree) [reply modular_zubbers/code/modules/clothing/shoes/boots.dm:215]: Otherwise, looking good.,

**Review by lessthnthree — CHANGES_REQUESTED** [review]

**Comment by Kingsley-95** [comment]
> * Is there a reason it's priced high as 6 TC? Looking at other job items it seems like it'd be more like 3.
 I can lower it, but due to someone being unable to remove the item themselves, I bumped it up a bit. I can lower it
>     * Is calling it 'slipers' instead of 'slippers' intentional?
It is, yeah. Mainly, to make it a clear difference between that and the actual shoe type of slippers. Idk if there ARE, but it felt more clear to me to have that distinction in the name

**Comment by NargaInDisguise** [comment]
> A idea might be is making it craftable out of bananium and clown shoes, it's pretty harmless

Absolutely not, makes you completely useless in combat, unless you make it a consumable recipe for a TC price. That could be fine. 8 tc for a clown to learn how to craft infinite bananium shoes...


Also make sure this is in the clown-ops uplink, shit'd be funny.

**Comment by Kingsley-95** [comment]
Not gunna make them craftable for now. A craftable version could be made that DOES allow the person who is wearing them to unequip themselves without delay. But not going to do that in this PR

**Review by lessthnthree — APPROVED** [review]

**Review by lessthnthree — CHANGES_REQUESTED** [review]
Needs a digi worn sprite and we're good to go.

**Pull request merged**